69 add capacity for footnotes in tables

sarraha786 opened the pull request:
I recovered the file and made the updates to the footnotes with Luca.

 - Closes #69 
 - Closes #67

kwheelan reviewed (changes requested):
@sarraha786 @lucakato Sorry I've been slow to look over this PR. I've consolidated the footnotes/readme changes from #69 and #94. Before we can merge, there are a couple of things to fix:

 - The final template needs to be called `ACFR_template.xlsx` because that's the file that the user downloads on the app. So, you'll need to save `ACFR_template_v2.xlsx` to `ACFR_template.xslx`.
 - The ReadMe looks good, but can you add something about the option to add Word documents at the upload stage of the app.
 - Generally, I want to make sure we've tested that this newer version of the template works with the app before merging this branch. This probably means filling out the newer template (unfortunately Marc's contacts filled out the old template), testing it on the app, and then making any adjustments to the template and code. Here are some issues I might anticipate:
 
### Footnotes
 - The Note needs to line up with column B or else the program will expect "Note" to be an xbrl tag. 
 - We want the program to ignore empty notes, so this will need to be a change in the Python.

### Custom lines
 - I like the idea of separating the custom fund balance line items in the GovFund balance sheet, but we'll need to test to make sure that title lines don't show up in the processed output. If so, we might need to adjust the Python to skip these lines.

lucakato commented:
@kwheelan thanks for this during busy times.
So we met on Thursday and @sarraha786 had a certain idea for modifying ACFR template so that the footnote 'Note:' message doesn't show on certain sheets, but now thinking about it doing this on Python might be easier?

Regarding testing by Marc's contacts, I still need to check this but I think they used the `main` branch code. So I need to run their templates against our latest dev code to see if it's different.

I've also just made a PR for a quick fix--realized page wasn't loading after conversion was complete.

kwheelan commented:
@lucakato Sounds good -- I think doing it in Python would be easier, but either way works

sarraha786 commented:
Hi, I addressed the edits for the readme and changing the name of the file,
but I ran into this message on my terminal and I wanted to run it by you
both before I do anything.


[image: Screen Shot 2024-10-01 at 11.50.05 AM.png]

On Mon, Sep 30, 2024 at 8:53 AM Katrina Wheelan ***@***.***>
wrote:

> @lucakato <https://github.com/lucakato> Sounds good -- I think doing it
> in Python would be easier, but either way works
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/closup/process-xbrl/pull/95#issuecomment-2383108005>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATD6TK6LNLIUPXUFKSNARR3ZZFCUBAVCNFSM6AAAAABOQ2DIBOVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDGOBTGEYDQMBQGU>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
*Sarrah Ahmed(She/Her)*
University of Michigan | Class of 2024
School of Information  | BSI
*248.971.4092 *| ***@***.*** | LinkedIn
<https://www.linkedin.com/in/sarrah-ahmed-b80861187/>

lucakato commented:
> Hi, I addressed the edits for the readme and changing the name of the file, but I ran into this message on my terminal and I wanted to run it by you both before I do anything. [image: Screen Shot 2024-10-01 at 11.50.05 AM.png]

the image doesn't show!

sarraha786 commented:
> Hi, I addressed the edits for the readme and changing the name of the file, but I ran into this message on my terminal and I wanted to run it by you both before I do anything. [image: Screen Shot 2024-10-01 at 11.50.05 AM.png]
> […](#)
> On Mon, Sep 30, 2024 at 8:53 AM Katrina Wheelan ***@***.***> wrote: @lucakato <https://github.com/lucakato> Sounds good -- I think doing it in Python would be easier, but either way works — Reply to this email directly, view it on GitHub <[#95 (comment)](https://github.com/closup/process-xbrl/pull/95#issuecomment-2383108005)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ATD6TK6LNLIUPXUFKSNARR3ZZFCUBAVCNFSM6AAAAABOQ2DIBOVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDGOBTGEYDQMBQGU> . You are receiving this because you were mentioned.Message ID: ***@***.***>
> -- *Sarrah Ahmed(She/Her)* University of Michigan | Class of 2024 School of
<img width="1182" alt="Screen Shot 2024-10-01 at 11 50 05 AM" src="https://github.com/user-attachments/assets/c45a3f0e-2f81-4bbf-a27e-484ad583c32f">
 Information | BSI *248.971.4092 *| ***@***.*** | LinkedIn <https://www.linkedin.com/in/sarrah-ahmed-b80861187/>



> > Hi, I addressed the edits for the readme and changing the name of the file, but I ran into this message on my terminal and I wanted to run it by you both before I do anything. [image: Screen Shot 2024-10-01 at 11.50.05 AM.png]
> 
> the image doesn't show!
<img width="1182" alt="Screen Shot 2024-10-01 at 11 50 05 AM" src="https://github.com/user-attachments/assets/9fecab88-32cb-4201-a4fc-342c17526e6b">

lucakato commented:
@sarraha786 have you tried `git pull`? what happens when you do `git status`?
also, I recommend using GitHub Desktop it'll make things easier
